extension/background.js

```diff
@@ -47,6 +51,18 @@ try {
 } catch (ex) {
     chromeVersion = 33;
 }
+try {
+    browser.runtime.getBrowserInfo().then(
```

Comment:
Replacing `chrome` by `browser` breaks the extension in Chrome because it is not available in Chrome. Need a workaround something like this:
```
var browser_api;
try {
	browser_api = browser;
    browser_api.runtime.getBrowserInfo().then(
        function(info) {
            if (info.name === 'Firefox') {
                firefoxVersion = info.version.replace(/[ab]\d+/, '');
            }
        }
    );
} catch (ex) {
    firefoxVersion = 0;
    browser_api = chrome;	// This is Chrome not Firefox
}
```
Then we need to use `browser_api` instead of `browser`. Same trick applies to in `popup.js`.